### DISCUSSION TITLE
How to access `cronworkflow.lastScheduledTime` from a Cron Workflow parameter

### DISCUSSION THREAD

**Discussion by MickeyShnaiderman-RecoLabs** [argoproj/argo-workflows#14010] [Q&A]
I am trying to inject the start and end timestamps as parameters in Cron Workflow. I want the timespan to be from the last scheduled time to the current time.

I tried the following with `cronworkflow.lastScheduledTime`: 
```
...
      - name: start-time
        value: {{=sprig.date("2006-01-02T15:04:05", cronworkflow.lastScheduledTime)}}
      - name: end-time
        value: {{=sprig.date("2006-01-02T15:04:05", workflow.creationTimestamp)}}
...
```

But it seems that only `workflow.creationTimestamp` is accessible. The `start-time` expression is being passed as is to the pod.

The Argo Workflow version I am using is v3.6.2.

**Comment by Joibel** [comment]
The `cronworkflow` variables are only available in the cronworkflow parameters, not in the workflow generated by the cronworkflow. I understand that this is confusing and in need of clarification, but right now the won't work in anything in below `spec.workflowSpec`
